lametro/templates/lametro/legislation.html

```diff
@@ -98,20 +97,25 @@ <h3 class="no-pad-bottom"><i class='fa fa-fw fa-list-ul'></i> History</h3>
                             </tr>
                         </thead>
                         <tbody>
-                            {% for action in actions %}
+                            {% for event in events %}
```

Comment:
I think `action` as a variable name may be a little bit more precise, since there can be more than one item corresponding to the same event (one being scheduled, with subsequent items for actions taken during the meeting). What do you think?